core/validation/failures.py

```diff
@@ -159,9 +213,30 @@ def __str__(self):
             f'Value "{self.value}" is not a valid enum value.'
         )
 
+    def uc_message_components(self) -> tuple[str, str, str, str]:
+        message = "You must select an option from the list provided. Do not populate the cell with your own content."
+        sheet, section = INTERNAL_TABLE_TO_FORM_LOCATION[self.sheet]
+        column = INTERNAL_COLUMN_TO_FORM_COLUMN[self.column]
```

Comment:
Could we use a line like 
```suggestion
        column = INTERNAL_COLUMN_TO_FORM_COLUMN.get(self.column) if not 'null' else self.column
```
No point mapping values onto themselves

Reply:
I think it's better to be verbose and add the somewhat redundant mappings in this case. I think this reduces readability for no efficiency gain.